xwiki-rendering-macros/xwiki-rendering-macro-box/src/main/java/org/xwiki/rendering/internal/macro/box/DefaultBoxMacro.java

```diff
@@ -59,7 +62,8 @@
      */
     public DefaultBoxMacro()
     {
-        super("Box", DESCRIPTION, new DefaultContentDescriptor(CONTENT_DESCRIPTION), BoxMacroParameters.class);
+        super("Box", DESCRIPTION, new DefaultContentDescriptor(CONTENT_DESCRIPTION, true,
+            new DefaultParameterizedType(null, List.class, Block.class)), BoxMacroParameters.class);
```

Comment:
Since `new DefaultParameterizedType(null, List.class, Block.class)` is a very common use case you should probably put it in a constant somewhere. Probably in Block.